feature_toggle_annotations.yaml

```diff
@@ -0,0 +1,30 @@
+# This code-annotations configuration file supports OEP-17, Feature Toggles.
+
+source_path: ./
+report_path: reports
+safelist_path: .annotation_safe_list.yml
+coverage_target: 50.0
+annotations:
+  documented_elsewhere:
+    - ".. documented_elsewhere:":
+    - ".. documented_elsewhere_name:":
+  feature_toggle:
+    - ".. toggle_name:":
+    - ".. toggle_type:":
+        choices: [switch, rollout, group]
```

Comment:
^ i think we want to have `waffle_flag` here as well (see: https://github.com/edx/edx-platform/blob/40ce96a8e3bdfc550c01ddfe74d79326d50d80f6/openedx/core/djangoapps/user_api/accounts/toggles.py#L10)